cmd/mswallet.go

```diff
@@ -62,23 +62,31 @@ var createmswalletcmd = &cobra.Command{
 			ExitWithError(err)
 		}
 
-		//register all wallets
-		err = registerMSWallets(wallets)
+		offline, err := cmd.Flags().GetBool("offline")
 		if err != nil {
-			ExitWithError(fmt.Sprintf("Error while registering ms sub wallets. The error is:\n %v\n", err))
+			fmt.Println("offline is not used or not set to true. Setting it to false")
 		}
 
-		groupWallet := wallets[0]
-		signerWallets := wallets[1:]
+		if !offline {
```

Comment:
we can return early by checking if offline is true. So 

```go
if offline {
  fmt.Println("....success..")
  return
}

// continue the registeration
```

Reply:
fixed.